### DISCUSSION TITLE
how to migrate all exsiting data to the new version?

### DISCUSSION THREAD

**Discussion by Ranko-Saotome** [open-webui/open-webui#787] [General]
Context, I'm updating to the latest version. It has a different name. I don't want to lose all my existing data and user accounts. In my honest opinion, there should be a way to auto migrate your data

**Comment by justinh-rahb** [comment]
When launching the new container just keep your previous docker volume name with `-v ollama-webui:/app/backend/data` in your `docker run` command.

**Comment by Ranko-Saotome** [comment]
Thanks!
